src/utils/fundamentals/fundamentals.js

```diff
@@ -0,0 +1,199 @@
+import * as utils from '../network/utils'
+import { searchData } from '../network'
+
+/**
+ * Make request and search data
+ * @function countries
+ * @param {object} params - object of parameters:
+ * @param {string} params.countryCode 
```

Comment:
You have called this parameter `countryCode` here but in the function itself it accepts `code`. So this should be `code` too.